ui-admin/src/study/workflow/WorkflowView.tsx

```diff
@@ -0,0 +1,336 @@
+import LoadingSpinner from 'util/LoadingSpinner'
+import CreateTriggerModal from '../notifications/CreateTriggerModal'
+import {
+  faAsterisk,
+  faCodeBranch, faEdit,
+  faTasks
+} from '@fortawesome/free-solid-svg-icons'
+import { faCalendarAlt, faClipboard, faEnvelope,   faCheckSquare } from '@fortawesome/free-regular-svg-icons'
+import InfoPopup from 'components/forms/InfoPopup'
+import _sortBy from 'lodash/sortBy'
+import { minutesToDayString, triggerName } from './workflowUtils'
+import { EllipsisDropdownButton } from 'components/forms/Button'
+
+/** shows configuration of notifications for a study */
+export default function WorkflowView({ studyEnvContext, portalContext }:
+  {studyEnvContext: StudyEnvContextT, portalContext: LoadedPortalContextT}) {
+  const currentEnv = studyEnvContext.currentEnv
+  const navigate = useNavigate()
+  const [triggers, setTriggers] = useState<Trigger[]>([])
+  const [studyEnvSurveys, setStudyEnvSurveys] = useState<StudyEnvironmentSurvey[]>([])
+  const [preEnrollSurvey, setPreEnrollSurvey] = useState<Survey>()
+  const [triggerOpts, setTriggerOpts] = useState<Partial<Trigger>>({})
+  const [previousEnv, setPreviousEnv] = useState<string>(currentEnv.environmentName)
+  const [showCreateModal, setShowCreateModal] = useState(false)
+
+  const boxClasses = 'p-3 my-3 bg-light'
+  const itemClasses = 'my-2 py-2'
+
+  const { isLoading, reload } = useLoadingEffect(async () => {
+    const [triggerList, surveyList] = await Promise.all([
+      Api.findTriggersForStudyEnv(portalContext.portal.shortcode,
+        studyEnvContext.study.shortcode, currentEnv.environmentName),
+      Api.findConfiguredSurveys(portalContext.portal.shortcode, studyEnvContext.study.shortcode,
+        currentEnv.environmentName, true, undefined)
+    ])
+    if (currentEnv.preEnrollSurveyId) {
+      const preEnrollSurvey = await Api.getSurveyById(portalContext.portal.shortcode, currentEnv.preEnrollSurveyId)
+      setPreEnrollSurvey(preEnrollSurvey)
+    }
+    setTriggers(triggerList)
+    setStudyEnvSurveys(_sortBy(surveyList, 'surveyOrder'))
+  }, [currentEnv.environmentName, studyEnvContext.study.shortcode])
+
+  useEffect(() => {
+    if (previousEnv !== currentEnv.environmentName) {
+      // the user has changed the environment -- we need to clear the id off the path if there
+      navigate(studyEnvWorkflowPath(paramsFromContext(studyEnvContext)))
+      setPreviousEnv(currentEnv.environmentName)
+    }
+  }, [currentEnv.environmentName])
+
+  const onCreate = (createdConfig: Trigger) => {
+    reload()
+    navigate(triggerPath(createdConfig, studyEnvContext.currentEnvPath))
+    setShowCreateModal(false)
+  }
+
+  const formEditLink = <Link to={studyEnvFormsParamsPath(paramsFromContext(studyEnvContext))}>
+    <FontAwesomeIcon icon={faEdit} className="ms-2 fa-xs fw-normal"/>
+  </Link>
+
+  return <div className="container-fluid px-4 py-2">
+    { renderPageHeader('Participant flow') }
```

Comment:
```suggestion
    { renderPageHeader('Participant Flow') }
```
nit: this and the sidebar link are a different casing from the rest of the sidebar/pages